packages/background/src/Pattern/dot.ts

```diff
@@ -0,0 +1,43 @@
+import { computed, defineComponent, h } from 'vue-demi'
+
+import { transformForCompatibleAttrs } from '../utils'
+
+interface DotPatternProps {
+  radius: number
+  color: string
+}
+
+const DotPattern = defineComponent({
+  name: 'DotPattern',
+  compatConfig: { MODE: 2 },
+  props: {
+    radius: {
+      type: Number,
+      default: () => 2,
+      required: false,
+    },
+    color: {
+      type: String,
+      default: () => '#eee',
+      required: false,
+    },
+  },
+  setup(props: DotPatternProps) {
+    const radius = computed(() => props.radius)
```

Comment:
This is wholly unnecessary btw, in multiple ways even

1. `computed` comes with caching, which produces overhead and therefore has better usage when it comes to expensive computations, not when returning a prop or doing simple equality comparisons like `prop.hello === 'world'`
Can be replaced by `toRef` which doesnt produce the same overhead 
2. Props are still reactive, even without wrapping them in extra reactive variables
Meaning if you do `return () => h('div', props.label)` and update the label prop, it will still re-render appropriately without wrapping `props.label` in `computed` or `toRef`.

Reply:
you Can  open a separate repository for Vue2